utils/transformISODates.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * utils/string/index.js
+ *
+ * Author: Caleb Panza
+ * Created: Feb 10, 2021
+ *
+ * String utility methods to be used throughout the platform
+ */
+
+import {
+  parseISO,
+  format,
+  isToday,
+  isWithinInterval,
+  addDays,
+  addYears,
+} from 'date-fns';
+
+/**
+ *
+ * @param {string} str  | String to be transformed
+ * @param {object} args |
+ */
+// eslint-disable-next-line import/prefer-default-export
+const transformISODates = (
+  str,
+  { isTodayText = 'Today', withTime = false } = {}
+) => {
+  if (!!str && typeof str === 'string') {
```

Comment:
Mentioned this before, and it's possibly personal preference, but I'm not a fan of big blocks of happy path code nested in an `if` statement — as opposed to a "bail out" condition like `if (!str || typeof str !== 'string') return null;`.

IMO it keeps the code tidier by keeping things flatter/less indented, and logically lets you read the code as inherently "on track" except for conditions where we must bail early.